doc/classes/EditorSelection.xml

```diff
@@ -30,12 +30,6 @@
 				Gets the list of selected nodes.
 			</description>
 		</method>
-		<method name="get_transformable_selected_nodes">
-			<return type="Array" />
-			<description>
-				Gets the list of selected nodes, optimized for transform operations (i.e. moving them, rotating, etc). This list avoids situations where a node is selected and also child/grandchild.
-			</description>
-		</method>
```

Comment:
Though this was not used internally, this could have been used externally by plugins, etc, so this probably shouldnt be deleted.